main/lsp/LSPTypechecker.h

```diff
@@ -92,10 +92,22 @@ class LSPTypechecker final {
         Cancelable,
     };
 
+    /**
+     * The result of a slow path operation depends on the mode that it was run in:
+     * - Init indicates that the LSPTypechecker is being initialized by the indexer. One implication of this is that the
+     *   slow path operation will not be cancelable, meaning that the result will always be committed. As we know that
+     *   the result will be committed, and the indexer needs a copy of the GlobalState to be taken after indexing, we
+     *   return that copy as the result of the slow path.
+     * - Cancelable indicates that the slow path may be canceled before it completes. As this mode does not require a
+     *   copy of the typechecker's GlobalState to be returned, and the slow path operation may be canceleld, we return a
```

Comment:
```suggestion
     *   copy of the typechecker's GlobalState to be returned, and the slow path operation may be canceled, we return a
```

Reply:
Incredible typo on my part, thank you for catching this!